orchestra/project_api/tests/test_project_api.py

```diff
@@ -32,7 +32,7 @@
 from orchestra.tests.helpers.fixtures import WorkflowVersionFactory
```

Comment:
The changes to tests largely revolve around the fact that we'd call the HTTP/Python APIs for `new_task_assignment` in various places to pick up a task for subsequent testing, and now that we removed those APIs, we more directly just call `assign_task`. In hindsight, this change was probably the better implementation all along, as it's a more direct way to get a task/worker in the correct configuration to run a test.